project/AppDependencies.scala

```diff
@@ -4,31 +4,23 @@ private object AppDependencies {
   import play.sbt.PlayImport
   import play.core.PlayVersion
 
-  private val playBootstrapVersion = "3.0.0"
-  private val domainVersion = "5.10.0-play-27"
-  private val hmrcTestVersion = "3.9.0-play-27"
-  private val akkaVersion = "2.5.26"
+  private val akkaVersion = "2.6.10"
 
   val compile = Seq(
-    "uk.gov.hmrc"              %% "mongo-lock"                % "6.23.0-play-27",
+    "uk.gov.hmrc"              %% "mongo-lock"                % "7.0.0-play-28",
     PlayImport.ws,
-    "uk.gov.hmrc"              %% "bootstrap-backend-play-27" % playBootstrapVersion,
-    "uk.gov.hmrc"              %% "domain"                    % domainVersion,
-    "com.typesafe.akka"        %% "akka-actor"                % akkaVersion,
-    "com.typesafe.akka"        %% "akka-testkit"              % akkaVersion,
-    "org.typelevel"            %% "cats-core"                 % "2.2.0",
+    "uk.gov.hmrc"              %% "bootstrap-backend-play-28" % "4.1.0",
     "org.reactivemongo"        %% "reactivemongo-iteratees"   % "0.18.8",
```

Comment:
Worth switching from reactivemongo to hmrc-mongo. Would be interesting to try out `akka-stream-alpakka-mongodb`

Reply:
alpakka isn't using akka 2.6 yet :(
Looking at the code though, the Mongo version is quite minimal, and we don't need it IMO:
`Source.fromPublisher` takes an Observable, which is what hmrc-mongo already gives us.

Reply:
Will be switching to hmrc-mongo in a later PR.